src/applications/model/gaming-streaming-server.cc

```diff
@@ -216,7 +220,7 @@ GamingStreamingServer::StartApplication (void)
     }
   m_socket->SetRecvCallback (MakeNullCallback<void, Ptr<Socket> > ());
   m_socket->SetAllowBroadcast (true);
-  for (size_t i = 0; i < m_pksList.size (); ++i)
+  for (size_t i = 0; i < m_packetSizeVariables.size (); ++i)
     {
       m_sendEvent[i] = Simulator::ScheduleNow (&GamingStreamingServer::Send, this, (int)i);
```

Comment:
You can make `GamingStreamingServer::Send` accept an uint type of variable instead of casting to int